Gruntfile.js

```diff
@@ -264,6 +264,11 @@ module.exports = function(grunt) {
         'karma:local'
     ]);
 
+    grunt.registerTask('ts', [
```

Comment:
I would prefer that this be added as a script task in `project.json`.

- `yarn ts` or `npm run ts` to run
- Less reliance on grunt
- We can still run script tasks in grunt by calling to `yarn` ci

Reply:
i would be down to add to `yarn type-check`